packages/react-combobox/src/components/Listbox/useListboxStyles.ts

```diff
@@ -1,24 +1,35 @@
+import { tokens } from '@fluentui/react-theme';
 import { SlotClassNames } from '@fluentui/react-utilities';
 import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
 import type { ListboxSlots, ListboxState } from './Listbox.types';
 
-/**
- * @deprecated Use `listboxClassNames.root` instead.
- */
-export const listboxClassName = 'fui-Listbox';
 export const listboxClassNames: SlotClassNames<ListboxSlots> = {
   root: 'fui-Listbox',
 };
 
+// TODO(sharing) use theme values once available
```

Comment:
Nit: You could put these constants in a shared place within this package to await being shared across fluent.